setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     psutil >= 5.4
     pybids >= 0.10.2
     pyyaml
+    requests
```

Comment:
at this point, why not just add `etelemetry` directly and leverage the check there:
`etelemetry.get_project("nipy/nipype")`

Reply:
I thought of using `_etrequest` directly, but we really don't want to do any kind of error handling. After going through the code it seemed more lightweight

Reply:
You bring up a good point... it would be useful to have an exception-free version in `etelemetry`.